src/main/java/com/prgrms/mukvengers/domain/store/model/Store.java

```diff
@@ -29,11 +38,59 @@ public class Store extends BaseEntity {
 	@GeneratedValue(strategy = IDENTITY)
 	private Long id;
 
+	@Convert(converter = PointConverter.class)
+	// @ColumnTransformer(write = "ST_PointFromText(?, 4326)", read = "ST_AsText(position)")
 	@Column(nullable = false)
 	private Point location;
 
+	@Column(nullable = false)
+	private String mapStoreId;
+
 	@Builder
-	protected Store(Point location) {
+	protected Store(Point location, String mapStoreId) {
+		validatePosition(location);
+		validateLongitude(location);
+		validateLatitude(location);
+		validateMapStoreId(mapStoreId);
+
```

Comment:
저는 이렇게 사용하는데 어떻게 생각하시나요..?
```suggestion
validateLocation(location);

public void validateLocation(Point location) {
		checkPosition(location);
		checkLongitude(location);
		checkLatitude(location);
	}
```